rules.d/90-iocost.rules

```diff
@@ -0,0 +1,20 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+SUBSYSTEM!="block", GOTO="iocost_end"
+
+ACTION=="remove", GOTO="iocost_end"
+
+IMPORT{builtin}="hwdb 'block:*:name:$env{ID_MODEL}:fwrev:$env{ID_REVISION}:'"
```

Comment:
This is backwards. Here you specify a "modalias string", and in the hwdb you have "match patterns" to match this strings. '*' can be used in "match patterns", but makes no sense to be used in the string that is being matched. 

(The confusion is may be caused by the fact that `abc:*:def` matches pattern `abc:*:def`, just like file `*.*` is matched by `*.*`. But still we don't use `*` in file names ;))

Please drop `:*` from the invocation here, and adjust the hwdb accordingly.

(If it is not possible to adjust the hwdb entries because too many have been released, then just replace `:*:` by `::` here.)